lib/stat_tracker.rb

```diff
@@ -110,7 +110,6 @@ def total_shots_per_team
     end
   end
```

Comment:
This `total_shots_per_team` method on line 106 is an old version of the `total_goals_per_team_per_season` and should be deleted.